opennlp-tools/src/main/java/opennlp/tools/util/DownloadUtil.java

```diff
@@ -80,6 +81,41 @@ public enum ModelType {
 
   private static Map<String, Map<ModelType, String>> availableModels;
 
+  /**
+   * Checks if a model of the specified {@code modelType} has been downloaded already
+   * for a particular {@code language}.
+   *
+   * @param language  The ISO language code of the requested model.
+   * @param modelType The {@link DownloadUtil.ModelType type} of model.
+   * @return {@code true} if a model exists locally, {@code false} otherwise.
+   * @throws IOException Thrown if IO errors occurred or the model is invalid.
+   */
+  static boolean existsModel(String language, ModelType modelType) throws IOException {
+    Map<ModelType, String> modelsByLanguage = getAvailableModels().get(language);
+    if (modelsByLanguage == null) {
+      return false;
+    } else {
+      final String url = modelsByLanguage.get(modelType);
+      if (url != null) {
+        final Path homeDirectory = Paths.get(System.getProperty("OPENNLP_DOWNLOAD_HOME",
```

Comment:
This is now across the `DownloadUtil`. Maybe we can just extract that into something like 

```java
private static Path getDownloadHome() { 
    return Paths.get(System.getProperty("OPENNLP_DOWNLOAD_HOME",
                System.getProperty("user.home"))).resolve(".opennlp");
}
```